vta/hardware/chisel/src/main/scala/core/TensorGemm.scala

```diff
@@ -39,10 +39,10 @@ class MAC(aBits: Int = 8, bBits: Int = 8, cBits: Int = 16) extends Module {
   val rA = RegNext(io.a)
   val rB = RegNext(io.b)
   val rC = RegNext(io.c)
-  
```

Comment:
These edits will add noise to the github history of the file. Can we fix all of the spacing issues in a separate PR for the Chisel codebase until we have lint tests?

Reply:
Actually I just realized - the edit was adding a tab; the spacing issues were already fixed before. That said a lint test would have caught that

Reply:
Sure, let me try to do that.

Btw, all the ASF headers for the entire codebase has the same issue.